App/Tools/libraries

```diff
@@ -79,3 +85,57 @@ function getInstalledVersion() : string
     }
     return 0;
 }
+
+/**
+ * Retourne le format long de la dernière mise à jour installée
+ * @exemple 1.12.4
+ */
+function getVersionDerniereMiseAJour(\includes\SQL $db, string $installedVersion) : string
+{
+    $req = 'SELECT appli_valeur FROM conges_appli WHERE appli_variable = "version_last_maj" LIMIT 1';
+    $res = $db->query($req);
+    $versionLastMaj = $res->fetch_array()['appli_valeur'];
+    $versionFile = file_get_contents('/tmp/lt-last-version');
```

Comment:
si le fichier n'existe pas, on a un warning : 

`PHP Warning:  file_get_contents(/tmp/lt-last-version): failed to open stream: Aucun fichier ou dossier de ce type in /web/App/Tools/libraries on line 98
`

Reply:
C'est #712. Je corrigerai, ce sera fait.